### DISCUSSION TITLE
Change the location of the build folder for SSR? (.output is not allowed by my host)

### DISCUSSION THREAD

**Discussion by notflip** [nuxt/nuxt#19151] [Questions] [answered]
Cyclic.sh doesn't allow the name of the dist folder to start with a "."
Is it possible to change the build folder location?

**Comment by AloisSeckar** [comment, marked as answer]
Sure it is. In Nuxt 3 it is part of Nitro config: https://nitro.unjs.io/config#output

Just for some reason I had to start my configured path with `../`...setting just `my-output` resulted into `/server/my-output/...` being created

```js
export default defineNuxtConfig({
  nitro: {
    output: {
      dir: '../my-output'
    }
  }
})
```

**Reply by notflip** [reply]
Aha! Thank you, I made a ticket regarding the issue with the `../`, since it was nesting it inside the server output like you stated above. Thank you!
